### PR TITLE
Translate hooks-state

### PR DESCRIPTION
**Pull request by a13marquez**

### CODE REVIEW

File: content/docs/hooks-state.md [hunk 1]
```diff
@@ -28,11 +29,11 @@ function Example() {
 }
 ```
 
-We'll start learning about Hooks by comparing this code to an equivalent class example.
```
Comment (alejandronanez) [inline content/docs/hooks-state.md]: What happened to this line?

File: content/docs/hooks-state.md [hunk 2]
```diff
@@ -28,11 +29,11 @@ function Example() {
...
 
-## Equivalent Class Example
 
-If you used classes in React before, this code should look familiar:
```
Comment (alejandronanez) [inline content/docs/hooks-state.md]: Also, what happened to this one?

File: content/docs/hooks-state.md [hunk 3]
```diff
@@ -56,15 +57,14 @@ class Example extends React.Component {
 }
 ```
 
-The state starts as `{ count: 0 }`, and we increment `state.count` when the user clicks a button by calling `this.setState()`. We'll use snippets from this class throughout the page.
```
Comment (alejandronanez) [inline content/docs/hooks-state.md]: I think we need to translate this

**Comment by carburo** [comment]
@a13marquez Yes, it is ok to translate comments, but not anything else inside code blocks.

File: content/docs/hooks-state.md (hunk 3 above)
Comment (a13marquez) [reply content/docs/hooks-state.md]: Fixed

File: content/docs/hooks-state.md (hunk 2 above)
Comment (a13marquez) [reply content/docs/hooks-state.md]: Fixed

File: content/docs/hooks-state.md (hunk 1 above)
Comment (a13marquez) [reply content/docs/hooks-state.md]: Fixed

File: content/docs/hooks-state.md [hunk 4]
```diff
@@ -56,39 +56,39 @@ class Example extends React.Component {
...
->Note
+>Nota
 >
->You might be wondering why we're using a counter here instead of a more realistic example. This is to help us focus on the API while we're still making our first steps with Hooks.
+>Puedes estar preguntndote porque estamos usando un contador en lugar de un ejemplo más realista. Esto es porque ayuda a centrarse en la API mientras seguimos dando nuestro primeros pasos on los Hooks
 
 ## Hooks and Function Components
 
-As a reminder, function components in React look like this:
+Como recordatorio, un componente funcional en React es así:
```
Comment (alejandronanez) [inline content/docs/hooks-state.md]:
```suggestion
Como recordatorio, un componente funcional en React se ve así:
```

**Pull request merged**